Remotion/ObjectBinding/Web/Globalization/BocList.de.resx

```diff
@@ -177,4 +177,7 @@
   <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.ValidationErrorInfoCell" xml:space="preserve">
     <value>Ungültige Eingabe in der aktuellen Zelle</value>
   </data>
+  <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.HasValidationFailureText" xml:space="preserve">
+      <value>Es wurden Fehler gefunden.</value>
```

Comment:
The problem is that we not only care for row errors, but errors in the  bocList itself aswell, while ListEditModeErrorMessage is only concerned with row errors.

@a-ctor Has a similar error field in RM-8866 however, which I think would be mergable with this one.

Reply:
I still think we can use the same message, perhaps separate IDs just in case. Aside from the list being empty (which has a dedicated error anyway) any error on the collection must imply errors within the lines. i.e. there is no obervable difference when talking about it.

Reply:
from the Localization task; we keep them separate but we need to check to suppress the EditModeValidator when using the new validation infrastructure.

Reply:
Closed because will be handled in RM-8875